LaTeX/README.md

```diff
@@ -1,25 +1,32 @@
 # LaTeX
```

Comment:
Didn't know about MD022, but I've been thinking about styling/linting for this project. Maybe we should look into setting it up as part of the CI process?

Reply:
I created #239 for this.

Reply:
Yea, I use VSC now, and have https://github.com/DavidAnson/vscode-markdownlint installed.